### DISCUSSION TITLE
What does "No destination for column XXXX" actually mean?

### DISCUSSION THREAD

**Discussion by claritydistro-nick** [stephenafamo/bob#165] [Q&A] [answered]
I have a relatively simple query. I'm receiving a relatively cryptic "No destination for column id" error and am now really sure how to debug.

```
 query := psql.Select(
    sm.From("users"),
    sm.Where(psql.Quote("users","id").EQ(user_id)),
    sm.LeftJoin("suppliers").OnEQ(
        psql.Quote("users","id"),
        psql.Quote("suppliers","ownerId"),
    )
)
```

**Comment by stephenafamo** [comment, marked as answer]
The issue would not be in the query creation but in scanning the results of the query.

The error means that when scanning the result, there is no where to scan the `id` column to.

**Reply by claritydistro-nick** [reply]
Got it. This is super helpful and thank you for the very quick reply!

**Comment by claritydistro-nick** [comment]
Thank
